fvtest/porttest/CMakeLists.txt

```diff
@@ -111,22 +109,31 @@ target_compile_options(sltestlib
 )
 
 if(OMR_HOST_OS STREQUAL "aix")
-	omr_add_library(aixbaddep SHARED
-		aixbaddep/sltest.c
+	omr_add_library(aixbaddep 
+		SHARED
+			aixbaddep/sltest.c
 	)
```

Comment:
nitpick, `SHARED` is a flag that doesn't accept any values. It should be up on the first line like this:
```cmake
	omr_add_library(aixbaddep SHARED
		aixbaddep/sltest.c
	)
```

Reply:
You can get rid of the extra tab as well